src/ReactTestRenderer.re

```diff
@@ -4,6 +4,7 @@ type t;
 external create: React.element => t = "create";
 
 [@mel.send] external toJSON: t => Js.Json.t = "toJSON";
+[@mel.send] external toObject: t => Js.t({.}) = "%identity";
```

Comment:
should probably be `Js.t({..})` to allow getting any key out with `##`?

Reply:
totally, fixed in 4940103.

Reply:
You forgot to change the interface file, which is where it really matters in this case .